aggregator/follower_flush_mgr.go

```diff
@@ -307,75 +319,41 @@ func (mgr *followerFlushManager) flushersFromKVUpdateWithLock(buckets []*flushBu
 }
 
 func (mgr *followerFlushManager) standardFlushersFromKVUpdateWithLock(
-	listID standardMetricListID,
+	resolution time.Duration,
 	flushers []flushingMetricList,
+	getFlushTimesByResolutionFn getFlushTimesByResolutionFn,
+	metrics standardFollowerFlusherMetrics,
+	logger log.Logger,
 ) []flusherWithTime {
 	var (
-		resolution       = listID.resolution
 		flushersWithTime = make([]flusherWithTime, 0, defaultInitialFlushCapacity)
 	)
 	for _, flusher := range flushers {
 		shard := flusher.Shard()
 		shardFlushTimes, exists := mgr.received.ByShard[shard]
 		if !exists {
-			mgr.metrics.standard.shardNotFound.Inc(1)
-			mgr.logger.WithFields(
-				log.NewField("shard", shard),
-			).Warn("shard not found in standard flusher flush times")
-			continue
-		}
-		lastFlushedAtNanos, exists := shardFlushTimes.StandardByResolution[int64(resolution)]
-		if !exists {
-			mgr.metrics.standard.resolutionNotFound.Inc(1)
-			mgr.logger.WithFields(
-				log.NewField("shard", shard),
-				log.NewField("resolution", resolution.String()),
-			).Warn("resolution not found in standard flusher flush times")
-			continue
-		}
-		newFlushTarget := flusherWithTime{
-			flusher:          flusher,
-			flushBeforeNanos: lastFlushedAtNanos,
-		}
-		flushersWithTime = append(flushersWithTime, newFlushTarget)
-		mgr.metrics.standard.kvUpdates.Inc(1)
-	}
-	return flushersWithTime
-}
-
-func (mgr *followerFlushManager) timedFlushersFromKVUpdateWithLock(
-	listID timedMetricListID,
-	flushers []flushingMetricList,
-) []flusherWithTime {
-	var (
-		resolution       = listID.resolution
-		flushersWithTime = make([]flusherWithTime, 0, defaultInitialFlushCapacity)
-	)
-	for _, flusher := range flushers {
-		shard := flusher.Shard()
-		shardFlushTimes, exists := mgr.received.ByShard[shard]
-		if !exists {
-			mgr.metrics.timed.shardNotFound.Inc(1)
-			mgr.logger.WithFields(
+			metrics.shardNotFound.Inc(1)
+			logger.WithFields(
```

Comment:
Include flush type here to distinguish standard metrics from timed metrics?

Reply:
The logger passed in already has that tag